src/zope/__init__.py

```diff
@@ -1 +1 @@
-__import__('pkg_resources').declare_namespace(__name__)
+__import__('pkg_resources').declare_namespace(__name__) # pragma: no cover
```

Comment:
That one is odd:  I'm guessing it gets skipped entirely by the namespace machinery, but then why does `coverage` pick it up at all?